### PR TITLE
update for 5.0.0

### PR DESCRIPTION
**Pull request by hexthat**

### CODE REVIEW

File: adafruit_rtttl.py [hunk 1]
```diff
@@ -40,6 +40,10 @@
     import audioio
     from adafruit_waveform import sine
     AUDIOIO_AVAILABLE = True
+    try:
+        import audiocore
+    except ImportError:
+        audiocore = audioio
```
Comment (ladyada) [inline adafruit_rtttl.py:46]: i think you want to do `import audioio as audiocore` intead of two imports?

**Review by ladyada — DISMISSED** [review]

File: adafruit_rtttl.py (hunk 1 above)
Comment (hexthat) [reply adafruit_rtttl.py:46]: I'm not sure what you mean.
Two imports are needed with 5.0.0 both audioio and audiocore.

also can the nrf stuff still use this or is extra code needed?

Comment (ladyada) [reply adafruit_rtttl.py:46]: ooh ok!

**Review by tannewt — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**